.github/workflows/build.yml

```diff
@@ -132,14 +129,15 @@ jobs:
       - name: Update Code Engine application
         run: |
           ibmcloud ce application update \
-          --name galasa-${{ matrix.region.name }}-site \
+          --name galasa-${{ matrix.region.name }}-${{ (github.ref == 'refs/heads/main') && 'site' || 'next'  }} \
```

Comment:
If you can't find a one-line solution, [this example](https://docs.github.com/en/actions/learn-github-actions/expressions#example-returning-a-json-object) looks like what we want to do.

Reply:
I' have corrected this and the Yaml syntax is working as expected.
@ind1go can you please review this again and approve if you are happy so that I can merge it in?